liberty-managed/src/main/java/io/openliberty/arquillian/managed/WLPManagedContainer.java

```diff
@@ -1405,10 +1409,17 @@ public String run() {
     * @throws IOException
     */
    private String getWlpUsrDir() throws IOException {
-      String usrDir = getLibertyEnvVar(WLP_USER_DIR);
-      if (usrDir == null) {
-         usrDir = containerConfiguration.getWlpHome() + "/usr/";
+      String usrDir;
+      if(containerConfiguration.getUsrDir() != null) {
+         usrDir = containerConfiguration.getUsrDir();
+      }
+      else {
+         usrDir = getLibertyEnvVar(WLP_USER_DIR);
+         if (usrDir == null) {
+            usrDir = containerConfiguration.getWlpHome() + "/usr/";
+         }
```

Comment:
Could simplify this a tiny bit:
String usrDir = containerConfiguration.getUsrDir();
if (usrDir == null) {
    usrDir = getLibertyEnvVar(WLP_USER_DIR);
....